File: java/problem-solving/Extra Problems/src/even_odd/EvenOdd.java
package even_odd;
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class EvenOdd {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Mir Labib Hossain(011183007)\nEnter array length : ");
        int i, n = Integer.parseInt(br.readLine());
        System.out.print("Enter " + n + " number : ");
        String s[] = br.readLine().split(" ");
        System.out.print("Even :");
        for (i = 0; i < n; i++) {
            if (Integer.parseInt(s[i]) % 2 == 0) {
                System.out.print(" " + s[i]);
            }
        }
        System.out.print("\nOdd :");
        for (i = 0; i < n; i++) {
            if (Integer.parseInt(s[i]) % 2 != 0) {
                System.out.print(" " + s[i]);
            }
        }
    }
}

File: c/problem-solving/Even Odd.c
#include <stdio.h>
int main()
{
    int i, n;
    printf("Enter the length of array : ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter %d number : ",n);
    for(i=0; i<n; i++)
    {
        scanf("%d",&arr[i]);
    }
    printf("Even : ");
    for(i=0; i<n; i++)
    {
        if(arr[i]%2==0)
            printf("%d, ",arr[i]);
    }
    printf("\nOdd : ");
    for(i=0; i<n; i++)
    {
        if(arr[i]%2!=0)
            printf("%d, ",arr[i]);
    }
    return 0;
}
